tests/unit/test_utilities.py

```diff
@@ -358,3 +361,53 @@ def test_make_array_two_dimensional(self):
 
         self.assertEqual(len(output_array.shape), 2)
         np.testing.assert_array_equal(output_array, expected_output)
+
+
+class TestTransposeIfXdimLessThanYdim(TestCase):
+
+    def test_wider_than_tall(self):
+        # Test case where x dim <  y dim and should transpose
+        input_array = np.ma.array([[1, 2, 3], [4, 5, 6]])
+        expected_output = np.ma.array([[1, 4], [2, 5], [3, 6]])
+        result = transpose_if_xdim_less_than_ydim(input_array)
+        np.testing.assert_array_equal(result, expected_output)
+        self.assertEqual(result.shape, (3, 2))
+
+    def test_taller_than_wide(self):
+        # Test case where x < y and should not transpose.
```

Comment:
I should know this, but is the standard that x stands for rows and y stands for columns? I've been assuming the x -> x-axis, y -> y-axis, but wanted to double check. @autydp also mentioned "across track" vs. "along track." Which three are equivalent?

Reply:
Actually, x is more usually the index for columns and y the index for rows.  In a swath, generally x is across-track and y is along-track. TEMPO seems to have this reversed, or at least the order of x and y are reversed in the array dimensions (1st, 2nd).